diff --git a/ext_drv/pcie/include/ax_pcie_dev.h b/ext_drv/pcie/include/ax_pcie_dev.h
index 9ff562586..e19f6352d 100644
--- a/ext_drv/pcie/include/ax_pcie_dev.h
+++ b/ext_drv/pcie/include/ax_pcie_dev.h
@@ -112,7 +112,7 @@
 } while (0)

 #if (LINUX_VERSION_CODE < KERNEL_VERSION(5, 17, 0))
-#define ax_pde_data(x)	PDE_DATA(x)
+#define ax_pde_data(x)	pde_data(x)
 #else
 #define ax_pde_data(x)	pde_data(x)
 #endif
diff --git a/ext_drv/pcie/net/rc-net/ax_pcie_net.c b/ext_drv/pcie/net/rc-net/ax_pcie_net.c
index 5a917650e..efdd3373c 100644
--- a/ext_drv/pcie/net/rc-net/ax_pcie_net.c
+++ b/ext_drv/pcie/net/rc-net/ax_pcie_net.c
@@ -1222,7 +1222,7 @@ static int __init axnet_pci_init_one(int idx)
 #endif

 #if (LINUX_VERSION_CODE < KERNEL_VERSION(6, 1, 0))
-	netif_napi_add(ndev, &axnet->napi, axnet_host_poll, AXNET_NAPI_WEIGHT);
+	netif_napi_add(ndev, &axnet->napi, axnet_host_poll);
 #else
 	netif_napi_add(ndev, &axnet->napi, axnet_host_poll);
 #endif
